Userland/Libraries/LibWeb/Painting/MediaPaintable.cpp

```diff
@@ -125,7 +125,8 @@ MediaPaintable::Components MediaPaintable::compute_control_bar_components(PaintC
     media_element.cached_layout_boxes({}).playback_button_rect = context.scale_to_css_rect(components.playback_button_rect);
     media_element.cached_layout_boxes({}).timeline_rect = context.scale_to_css_rect(components.timeline_rect);
     media_element.cached_layout_boxes({}).speaker_button_rect = context.scale_to_css_rect(components.speaker_button_rect);
-    media_element.cached_layout_boxes({}).volume_rect = context.scale_to_css_rect(components.volume_rect);
+    media_element.cached_layout_boxes({}).volume_click_rect = context.scale_to_css_rect(components.volume_rect);
+    media_element.cached_layout_boxes({}).volume_rect = context.scale_to_css_rect(components.volume_rect.shrunken(components.volume_button_size, 0));
```

Comment:
Good eye!

The effect of this change is a clear improvement, but these lines read a bit awkwardly to me. I'm reading this as "the volume_click_rect is the volume_rect, and then the volume_rect is a section of the volume_rect", so it feels a bit confusing.

Can we do something like this? The diff is a tiny bit more noisy, but I think it makes it a bit clearer what each distinct rect is meant to achieve:
https://gist.github.com/trflynn89/e0a48b0048d7af919e593d8cb7a69a58

Reply:
Looks good to me, I've added that to it!